dotnet/src/webdriver/Safari/SafariDriver.cs

```diff
@@ -61,6 +62,10 @@ namespace OpenQA.Selenium.Safari
     /// </example>
     public class SafariDriver : WebDriver
     {
+        private const string OpenInspector = "attachDebugger";
```

Comment:
Suffix each of these constants with `Command` (e.g., `OpenInspectorCommand`, `GetPermissionsCommand`, etc.).

Reply:
This definitely makes the naming less awkward.